doc/conf.py

```diff
@@ -469,6 +469,15 @@ def add_js_css_files(app, pagename, templatename, context, doctree):
     "auto_examples/exercises/plot_cv_digits.py": (
         "auto_examples/model_selection/plot_nested_cross_validation_iris.py"
     ),
+    "auto_examples/cluster/plot_agglomerative_clustering.py": (
```

Comment:
I would not remove this example: one is discussing the `linkage` and its insight while the second discuss the effect of a metric that is a completely different topic.